src/next/components/InfoCols/InfoCols.tsx

```diff
@@ -0,0 +1,77 @@
+import { BiRightArrowAlt } from "react-icons/bi"
+import { InfoColsProps } from "~/common"
+import { SUPPORTED_ICONS_MAP, SupportedIconName } from "~/common/Icons"
+
+const InfoColsHeader = ({
+  title,
+  subtitle,
+}: Pick<InfoColsProps, "title" | "subtitle">) => (
+  <div className="w-full flex flex-col gap-7 items-start text-left">
+    <h1 className="text-[#333333] text-4xl font-normal leading-tight">
+      {title}
+    </h1>
+    {subtitle && <p className="text-xl font-normal">{subtitle}</p>}
+  </div>
+)
+
+const InfoBoxIcon = ({ icon }: { icon?: SupportedIconName }) => {
+  if (!icon) return null
+  const Icon = SUPPORTED_ICONS_MAP[icon]
+  return (
+    <div>
+      <Icon className="w-10 h-auto text-secondary" />
+    </div>
+  )
+}
+
+const InfoBoxes = ({ infoBoxes }: Pick<InfoColsProps, "infoBoxes">) => {
+  return (
+    <div className="grid grid-cols-1 gap-x-8 gap-y-12 md:grid-cols-2 xl:grid-cols-3">
+      {infoBoxes.map((infoBox, idx) => (
+        <div key={idx} className="flex flex-col gap-4 items-start text-left">
+          <InfoBoxIcon icon={infoBox.icon} />
+          <div className="flex flex-col gap-1 items-start text-[#2C2E34]">
+            <h3 className="font-bold">{infoBox.title}</h3>
+            <p>{infoBox.description}</p>
+          </div>
+          {infoBox.buttonLabel && infoBox.buttonUrl && (
+            <a
```

Comment:
Might need to use the LinkComponent instead that I merged recently

Reply:
[4da0053](https://github.com/isomerpages/isomer-components/pull/10/commits/4da005345aae3ed41794a3a59388d1259f9887e8)